encoding/openapi/openapi.go

```diff
@@ -81,6 +81,12 @@ type Config struct {
 	// OpenAPI Schema. It is an error for an CUE value to refer to itself
 	// if this option is used.
 	ExpandReferences bool
+
+	// MaxCycleDepth is a value that adds a maximum of iterations to stop evaluating
+	// structure cycles. The current behaviour does not have any stop condition, and it could
+	// generate infinite loops working with recursive values.
+	// The value should be big enough to fix the desire use case, but it must be used carefully.
```

Comment:
just some suggested doc tweaks

```suggestion
	// MaxCycleDepth specifies the maximum number of steps to search for structural
	// cycles.
	//
	// This is a hack, not a general purpose solution to cycle detection. Use it
	// with care. Set a value large enough to capture known use cases.
	//
	// NOTE This option does not exist in canonical cue-lang/cue.
```